improver/calibration/simple_bias_correction.py

```diff
@@ -0,0 +1,300 @@
+    if collapse_dim in get_dim_coord_names(forecast_errors):
+        mean_forecast_error = collapsed(
+            forecast_errors, collapse_dim, iris.analysis.MEAN
+        )
+        return mean_forecast_error.data
+    return forecast_errors.data
+
+
+def apply_additive_correction(
+    forecast: Cube, bias: Cube, fill_masked_bias_values: bool = True
+) -> ndarray:
+    """
+    Apply additive correction to forecast using the specified bias values,
+    where the bias is expected to be defined as forecast - truth.
+
+    Args:
+        forecast:
+            Cube containing the forecast to which bias correction is to be applied.
+        bias:
+            Cube containing the bias values to apply to the forecast.
+
+    Returns:
+        An array containing the corrected forecast values.
+    """
+    if fill_masked_bias_values and isinstance(bias.data, ma.masked_array):
+        bias.data = ma.MaskedArray.filled(bias.data, 0.0)
+    corrected_forecast = forecast - bias
+    return corrected_forecast.data
+
+
+class CalculateForecastBias(BasePlugin):
+    """
+    A plugin to evaluate the forecast bias from the historical forecast and truth
+    value(s).
+    """
+
+    def __init__(self):
+        """
+        Initialise class for calculating forecast bias.
+        """
+        self.error_method = evaluate_additive_error
+
+    def _define_metadata(self, forecasts: Cube) -> Dict[str, str]:
+        """
+        Define metadata for forecast bias cube, whilst ensuring any mandatory
+        attributes are also populated.
+
+        Args:
+            forecasts:
+                The source cube from which to get pre-existing metadata.
+
+        Returns:
+            A dictionary of attributes that are appropriate for the bias cube.
+        """
+        attributes = generate_mandatory_attributes([forecasts])
+        attributes["title"] = "Forecast bias data"
+        return attributes
+
+    def _create_bias_cube(self, forecasts: Cube) -> Cube:
+        """
+        Create a cube to store the forecast bias data.
+
+        Where multiple reference forecasts values are provided via forecasts,
+        the time dimension will be collapsed to a single value represented by
+        a single forecast_reference_time with bounds set using the range of
+        forecast_reference_time values present in forecasts.
+
+        Args:
+            forecasts:
+                Cube containing the reference forecasts to use in calculation
+                of forecast bias.
+
+        Returns:
+            A copy of the forecasts cube with the attributes updated to reflect
+            the cube is the forecast error of the associated diagnostic. If a time
+            dimension is present in the forecasts, this will be collapsed to a single
+            value.
+        """
+        attributes = self._define_metadata(forecasts)
+        forecast_bias_cube = create_new_diagnostic_cube(
+            name=f"{forecasts.name()}_forecast_error",
```

Comment:
I notice that Rainforests and ensemble_copula_coupling uses the name `forecast_error_of_*` rather than `*_forecast_error`.

Reply:
Thanks @nivnac. I had a chat with Tim around what cf-conventions say we should do on this, but failed to look back over what we had used for rainforests work. I think being consistent is important, so I'll update the variable naming across the PR.

Reply:
Also, do you think we need any other metadata or attributes?

Reply:
None that I can think of.

Reply:
Great, thanks James! I'm just updating the acceptance test data and then I think this should be ready for MO review.